lib/classes/savings_account.rb

```diff
@@ -0,0 +1,28 @@
+require 'awesome_print'
+module BankAccount
+  class Savings < Account
+
+    def initialize(id, initial_balance)
+      super
+      if @current_balance < 10
+        raise ArgumentError.new, "Initial deposit must be greater than 10"
+      end
+    end
+
+    def withdraw(amount)
+      super
+      if amount > (@current_balance - 10)
+        raise ArgumentError.new, "Savings cannot go below 10"
+      end
+      return (@current_balance -= 2)
+    end
+
+    def add_interest(rate)
```

Comment:
Haha! I think I was paranoid about when and when not to use them because I didn't understand the difference very well, so I just defaulted to using them always forever. 
I have a better idea now of when and when not to use an instance variable. :)